tests/Feature/SearchTest.php

```diff
@@ -14,7 +14,7 @@
 
 final class SearchTest extends IntegrationTestCase
 {
-    public function test_search_with_filters(): void
```

Comment:
You can test both where and wherein in one test. I prefer less acceptance tests. Low level logic should be tested in unit tests

Reply:
I merged the two tests into one as requested